### PR TITLE
pkp/pkp-lib#1739 theme options

### PR DESCRIPTION
**Pull request by NateWr**

### CODE REVIEW

**Comment by bozana** [comment]
I had a roughly code review -- it looks good to me

File: plugins/themes/default/DefaultThemePlugin.inc.php [hunk 1]
```diff
@@ -17,17 +17,10 @@
 
 class DefaultThemePlugin extends ThemePlugin {
 	/**
-	 * Constructor
-	 */
-	function DefaultThemePlugin() {
```
Comment (bozana) [inline plugins/themes/default/DefaultThemePlugin.inc.php]: Is the constructor not necessary?

Comment (asmecher) [inline plugins/themes/default/DefaultThemePlugin.inc.php]: Generally I've been including them. Might be worth reviewing at some point, both to make inclusion/exclusion consistent, and probably to change over to newer PHP syntax, but ... later. I think these used to be strictly necessary, way back when in PHP4 days, if you wanted parent class constructors to be called.

Comment (bozana) [inline plugins/themes/default/DefaultThemePlugin.inc.php]: OK

File: plugins/themes/default/DefaultThemePlugin.inc.php [hunk 2]
```diff
@@ -17,17 +17,10 @@
...
-		parent::ThemePlugin();
-	}
-
-	/**
 	 * @copydoc ThemePlugin::isActive()
 	 */
 	public function isActive() {
-                if (defined('SESSION_DISABLE_INIT')) return true;
+        if (defined('SESSION_DISABLE_INIT')) return true;
```
Comment (asmecher) [inline plugins/themes/default/DefaultThemePlugin.inc.php]: Accidental change from tabs to spaces.

**Comment by NateWr** [comment]
Tests failed when trying to find a logout link. 

https://travis-ci.org/pkp/omp/jobs/156501742#L819

I've restarted to see if we get lucky.

**Pull request merged**